### PR TITLE
JBPM-7002 - Group action buttons in list table to kebab

### PR DESCRIPTION
**Pull request by cristianonicolai**

### CODE REVIEW

**Comment by cristianonicolai** [comment]
@nmirasch @bsremac please take a look.

**Comment by cristianonicolai** [comment]
Depends on https://github.com/kiegroup/appformer/pull/234

**Comment by bsremac** [comment]
@cristianonicolai I'm getting these errors when building _jbpm-wb_:
```
[ERROR] Errors in 'jar:file:/home/bsremac/.m2/repository/org/kie/workbench/screens/kie-wb-common-library-client/7.7.0-SNAPSHOT/kie-wb-common-library-client-7.7.0-SNAPSHOT-sources.jar!/org/kie/workbench/common/screens/library/client/util/LibraryPlaces.java'
[INFO]          [ERROR] Line 96: ALERTS cannot be resolved or is not a field
[INFO]    Tracing compile failure path for type 'org.jbpm.designer.client.DesignerPresenter'
[INFO]       [ERROR] Errors in 'jar:file:/home/bsremac/.m2/repository/org/jbpm/jbpm-designer-client/7.7.0-SNAPSHOT/jbpm-designer-client-7.7.0-SNAPSHOT-sources.jar!/org/jbpm/designer/client/DesignerPresenter.java'
[INFO]          [ERROR] Line 160: AlertsButtonMenuItemBuilder cannot be resolved to a type
[INFO]       [ERROR] Errors in 'jar:file:/home/bsremac/.m2/repository/org/kie/workbench/widgets/kie-wb-metadata-widget/7.7.0-SNAPSHOT/kie-wb-metadata-widget-7.7.0-SNAPSHOT-sources.jar!/org/kie/workbench/common/widgets/metadata/client/KieEditor.java'
[INFO]          [ERROR] Line 31: The import org.guvnor.messageconsole.client.console.widget.button cannot be resolved
[INFO]          [ERROR] Line 89: AlertsButtonMenuItemBuilder cannot be resolved to a type
[INFO]          [ERROR] Line 282: AlertsButtonMenuItemBuilder cannot be resolved to a type
...
```

**Comment by cristianonicolai** [comment]
@bsremac rebased PR  kiegroup/appformer#234, should be fine now.

File: jbpm-wb-common/jbpm-wb-common-client/src/main/java/org/jbpm/workbench/common/client/util/ConditionalAction.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    private Boolean navigational;
+
+    public ConditionalAction(final String text,
+                             final Consumer<T> callback,
+                             final Predicate<T> predicate,
+                             final Boolean navigational) {
+        this.predicate = predicate;
+        this.text = text;
+        this.callback = callback;
+        this.navigational = navigational;
```
Comment (bsremac) [inline jbpm-wb-common/jbpm-wb-common-client/src/main/java/org/jbpm/workbench/common/client/util/ConditionalAction.java]: isNavigation?

**Review by bsremac — APPROVED** [review]
Looks good. One minor comment, which I was reluctant to even write because the word made me laugh :)

File: jbpm-wb-common/jbpm-wb-common-client/src/main/java/org/jbpm/workbench/common/client/util/ConditionalAction.java (hunk 1 above)
Comment (cristianonicolai) [reply jbpm-wb-common/jbpm-wb-common-client/src/main/java/org/jbpm/workbench/common/client/util/ConditionalAction.java]: kkkk ok, will get that fixed :P

**Pull request merged**